widget/embedded/src/hooks/useBootstrap/useBootstrap.ts

```diff
@@ -69,4 +87,50 @@ export function useBootstrap() {
 
     return tabManager.destroy;
   }, []);
+
+  useEffect(() => {
```

Comment:
I'm not sure if deleting WidgetEvents and moving it to useBootstrap is a good idea.  The previous version was capsulated and did just one thing.